.travis.yml

```diff
@@ -13,12 +14,15 @@ script:
   - cargo build --verbose
   - cargo test --verbose
   - cargo clippy --all-targets --all-features -- -D warnings
+jobs:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+
 matrix:
   include:
     - rust: nightly
```

Comment:
We seem to be building nightly twice for Linux.

I'm really not familiar with Travis, but maybe if we add an `os` entry under this `include` with both operating systems it should work? (the -rust part is repeated, so I'm just guessing if it works the same)
We should probaby also remove the new `jobs` entry, but readding the `allow_failures` part under `matrix`.

BTW you can test this against your own fork if you want until it works, I did it last time I had to work on this and I preferred it :)